src/features/steps/verify/Verification.tsx

```diff
@@ -141,31 +154,25 @@ export default () => {
                     )}
                 </div>
                 {failed && (
-                    <>
-                        <br />
-                        <br />
-                        <p>Could not communicate with the kit.</p>
-                        <br />
-                        <p>
-                            Contact support on <DevZoneLink /> if the problem
-                            persist.
-                        </p>
-                    </>
+                    <div className="tw-pt-8">
+                        <NoticeBox
+                            mdiIcon="mdi-lightbulb-alert-outline"
```

Comment:
Not my field of expertise, so just a "user feedback": The icon here is not immediate to me. If the errors are about connection, maybe a red plug would be better? Or the same lightning icon as for programming?

Reply:
I guess in this case the user doesn't really know that something is/may be wrong with the connection.

@ketile any comments or should we keep the lightbulb in the verify step?